fastlane/lib/fastlane/documentation/markdown_docs_generator.rb

```diff
@@ -82,6 +82,12 @@ def all_actions_from_enhancer
       require 'faraday'
       require 'json'
 
+      if ENV['USE_ENHANCE_TEST_DATA']
```

Comment:
Also, please add a short comment above here please as to why this is needed (e.g. only Fabric team members have access to the enhancer instance, so this can be used to check docs changes for everyone else - or something like that)